molgenis-searchall/src/main/resources/l10n/searchall_nl.properties

```diff
@@ -0,0 +1,14 @@
+search-placeholder=Zoek in alle data
+search-button-label=Zoek
+clear-button-label=Opnieuw
```

Comment:
Doet mij denken dat ik opnieuw kan zoeken, maar leegt in plaats daarvan het zoekveld

Reply:
Was er zelf ook niet helemaal tevreden mee, maar "leegmaken" vond ik ook niet helemaal mooi staan... dus wat wel...

Reply:
Ik denk Reset of Leegmaken